pilot/pkg/model/jwks_resolver.go

```diff
@@ -285,15 +316,15 @@ func (r *jwksResolver) refresher() {
 	r.refreshTicker = time.NewTicker(r.refreshInterval)
 	for {
 		select {
-		case now := <-r.refreshTicker.C:
-			r.refresh(now)
+		case <-r.refreshTicker.C:
```

Comment:
What is the effect of this change?

Reply:
The refresh job will try to update the public key independently from the expire time.